conformance/lang/values-types-and-variables/string.balt

```diff
@@ -0,0 +1,64 @@
+Test-case: output
+Description: Test valid string values
+Label: string
```

Comment:
For each type `T` I thought we discussed including error tests that 
1. assign invalid values to a variable of type `T` and 
2. assign values that belong to `T` to variables of other types

Reply:
Missed that we have a few in L46-L50. 

Shouldn't they be separate tests though? The description "Test invalid string values" isn't quite accurate for L50 (`int f = "Qwerty";`).